src/app/shared/search/search.utils.ts

```diff
@@ -49,7 +49,7 @@ export function stripOperatorFromFilterValue(value: string) {
  * @param operator
  */
 export function addOperatorToFilterValue(value: string, operator: string) {
-  if (!value.match(new RegExp(`^.+,(equals|query|authority)$`))) {
+  if (!value.match(new RegExp(`^.+,(equals|query|authority|contains|notcontains|notequals)$`))) {
```

Comment:
This doesn't appear to be related to this PR?  This appears to be accidentally copied over from another PR (perhaps #2608).  So, I think this can be removed from this PR.

Reply:
Again, this code change doesn't appear to be related to this PR? This appears to be accidentally copied over from another PR (perhaps #2608). So, I think this can be removed from this PR.

Reply:
remove here